### PR TITLE
Fix typo Esi in part create framework

### PR DESCRIPTION
**Pull request by nicolasdewez**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets |  |

Name for Esi class was wrong : ESI -> Esi

### CODE REVIEW

**Comment by OskarStark** [comment]
:+1:  you are right

```php
<?php

/*
 * This file is part of the Symfony package.
 *
 * (c) Fabien Potencier <fabien@symfony.com>
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */

namespace Symfony\Component\HttpKernel\HttpCache;

use Symfony\Component\HttpFoundation\Request;
use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\HttpKernel\HttpKernelInterface;

/**
 * Esi implements the ESI capabilities to Request and Response instances.
 *
 * For more information, read the following W3C notes:
 *
 *  * ESI Language Specification 1.0 (http://www.w3.org/TR/esi-lang)
 *
 *  * Edge Architecture Specification (http://www.w3.org/TR/edge-arch)
 *
 * @author Fabien Potencier <fabien@symfony.com>
 */
class Esi implements SurrogateInterface
{
    private $contentTypes;
...
```

**Comment by weaverryan** [comment]
:+1:

File: create_framework/http-kernel-httpkernelinterface.rst [hunk 1]
```diff
@@ -153,7 +153,7 @@ sub-requests to convert them to their proper content::
     $framework = new HttpKernel\HttpCache\HttpCache(
         $framework,
         new HttpKernel\HttpCache\Store(__DIR__.'/../cache'),
-        new HttpKernel\HttpCache\ESI()
+        new HttpKernel\HttpCache\Esi()
```
Comment (xabbuh) [inline create_framework/http-kernel-httpkernelinterface.rst]: Shouldn't we also fix the namespace (the `Symfony\Component` part is missing and there is no `use` statement for it)?

Comment (WouterJ) [inline create_framework/http-kernel-httpkernelinterface.rst]: let's merge this PR as-is

**Comment by WouterJ** [comment]
Wow, this error isn't even discovered by PHP, but you found it and fixed it. Thanks!

**Pull request closed without merging**